app/controllers/v0/sessions_controller.rb

```diff
@@ -213,5 +214,14 @@ def context_key
     rescue
       'unknown'
     end
+
+    def benchmark_tags(success = true)
```

Comment:
minor nit, but I'd prefer to see this parameter as either a k/v pair or `*tags` that can be added to the tags array in addition to the context, loa and multifactor tags.

Reply:
I'm not sure I'm following 100% here. Can you give more detail?

Reply:
```
def benchmark_tags(*tags)
      tags << "context:#{context_key}"
      tags << "loa:#{@current_user.loa[:current]}" if @current_user
      tags << "multifactor:#{@current_user.multifactor}" if @current_user
      tags
end
```

and then

`benchmark_tags('status:success')`/`benchmark_tags('status:failure')`.

Reply:
Ok, I like that better. Will update